static/css/timeline.scss

```diff
@@ -289,4 +289,12 @@ sup {
   margin: 10px;
   font-size: 12px;
   color: #666;
+}
+
+#timeline-lower-pane {
+  display: flex;
+}
+
+#download-link {
+  color: #fff;
```

Comment:
This shouldn't be required... btn-dark already gives it white text. Can you try removing and see?

Reply:
removed and changed to use <button>